docs/nulecule.md

```diff
@@ -0,0 +1,242 @@
+# Nulecule file
+
+A lot of information here is copied from the [Nulecule specification](https://github.com/projectatomic/nulecule/tree/master/spec).
+
+There are __minor__ differences as we implement features not yet implemented into `atomicapp` from the specification. Please refer to this reference for deploying your application.
+
+A `Nulecule` file format can either be `json`, `yaml`, or `ini`.
```

Comment:
Yeah... that's wrong >.>. Woops. It should be just json or yaml. Was thinking about answers.conf file generation when I typed that.